simple/single-script/create/funcs

```diff
@@ -214,7 +214,7 @@ function progress_bar(){
 
       if [[ $time_proc -gt 250 ]]
       then
-          echoBold "\n\nSomething went wrong! Please Follow < FAQ-Link > for more information\n"
+          echoBold "\n\nSomething went wrong! Please Follow \"https://docs.google.com/document/d/1BqRmTFu8qJG32g018mgpdgmmLuhJDDK1WgJhnL5hYe8/edit?usp=sharing\" for more information\n"
```

Comment:
@NishikaDeSilva Shouldn't we include these FAQs at the official website [1] rather than maintaining them in a Google Document?

[1]: [WSO2 official website](https://wso2.com/)

Reply:
yeah. Since it was not published at the time of release, I put a google doc to the others. I have missed this somehow

Reply:
Can you please update this link in such a way that it points to the website FAQ?